packages/tool-cache/__tests__/tool-cache.test.ts

```diff
@@ -1,8 +1,8 @@
 import * as fs from 'fs'
-import * as nock from 'nock'
 import * as path from 'path'
 import * as io from '@actions/io'
 import * as exec from '@actions/exec'
+import nock from 'nock'
```

Comment:
reaction due to moving `"esModuleInterop": true` to the root `tsconfig.json` - which previously was only defined in `packages/github/tsconfig.json`